package.json

```diff
@@ -2,6 +2,7 @@
   "name": "angular-ui-tree",
   "version": "2.7.0",
   "description": "An AngularJS UI component that can sort nested lists, provides drag & drop support and doesn't depend on jQuery",
+  "main": "index.js",
```

Comment:
Is it possible to use `./dist/angular-ui-tree` directly as a value for the main file? Then we don't have to create an extra `index.js` file.